src/main/java/org/sonarlint/intellij/config/global/SonarQubeServerEditor.java

```diff
@@ -223,6 +227,14 @@ protected ValidationInfo doValidate() {
     return rootPanel;
   }
 
+  private void openOrganizationList(ActionEvent e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "e". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1172)